lib/l10n/app_en.arb

```diff
@@ -443,6 +443,30 @@
   "@list": {},
   "grid": "Grid",
   "@grid": {},
+  "customizationSettingsTitle": "Customization",
+  "@customizationSettingsTitle": {
+    "description": "Title for the customization settings screen"
+  },
+  "playbackSpeedControlSetting": "Playback Speed Visibility",
+  "@playbackSpeedControlSetting": {},
+  "playbackSpeedControlSettingSubtitle": "Whether the playback speed controls are shown in the player screen menu",
+  "@playbackSpeedControlSettingSubtitle": {},
+  "automatic": "Automatic",
+  "@automatic": {},
+  "shown": "Shown",
+  "@shown": {},
+  "hidden": "Hidden",
+  "@hidden": {},
```

Comment:
Please add descriptions that explain how and where these tokens are used, and (if necessary) how they should be worded